nomos-blend/message/Cargo.toml

```diff
@@ -15,10 +15,10 @@ groth16         = { workspace = true }
 itertools       = { default-features = false, version = "0.14" }
 nomos-core      = { workspace = true }
 nomos-utils     = { features = ["rng"], workspace = true }
-num-bigint      = { version = "0.4", default-features = false }
+num-bigint      = { default-features = false, version = "0.4" }
 poq             = { workspace = true }
 serde           = { default-features = false, version = "1" }
-serde-big-array = { version = "0.5", default-features = false }
+serde-big-array = { default-features = false, version = "0.5" }
```

Comment:
are these cargo toml changes from a lint? curious why it's showing up only on your PR?

Reply:
ok, then lets just revert these if it's not meaningful for the PR.

You can use this command to do the revert on each file
```
git checkout master -- nomos-blend/messages/Cargo.toml
```